cesium/channel_test.go

```diff
@@ -118,6 +119,108 @@ var _ = Describe("Channel", func() {
 					f := MustSucceed(db.Read(ctx, telem.TimeRangeMax, key))
 					Expect(f.Series[0].Data).To(Equal(telem.NewSeriesV[int64](1, 2, 3, 4, 5).Data))
 				})
+
+				It("Should error if the meta.json file is corrupted", func() {
+					s := MustSucceed(fs.Sub("sub1"))
+					db := MustSucceed(cesium.Open("", cesium.WithFS(s)))
+					key := GenerateChannelKey()
+
+					Expect(db.CreateChannel(ctx, cesium.Channel{Key: key, Rate: 1 * telem.Hz, DataType: telem.Int64T})).To(Succeed())
+					Expect(db.Close()).To(Succeed())
+
+					f, err := s.Open(strconv.Itoa(int(key))+"/meta.json", os.O_WRONLY)
+					Expect(err).ToNot(HaveOccurred())
+					_, err = f.Write([]byte("heheheha"))
+					Expect(err).ToNot(HaveOccurred())
+					Expect(f.Close()).To(Succeed())
+
+					db, err = cesium.Open("", cesium.WithFS(s))
+					Expect(err).To(MatchError(ContainSubstring("error decoding meta file")))
+				})
+
+				It("Should error if the meta.json file got changed with impossible configurations", func() {
```

Comment:
This should be a table spec no?

Reply:
Also, wouldn't it be possible to move these specs into the `meta` package instead?